dll/win32/shell32/wine/shellord.c

```diff
@@ -345,36 +345,51 @@ int ShellMessageBoxW(
 	UINT uType,
 	...)
 {
-	WCHAR	szText[100],szTitle[100];
-	LPCWSTR pszText = szText, pszTitle = szTitle;
-	LPWSTR  pszTemp;
-	__ms_va_list args;
-	int	ret;
+    WCHAR *szText = NULL, szTitle[100];
+    LPCWSTR pszText, pszTitle = szTitle;
+    LPWSTR pszTemp;
+    __ms_va_list args;
```

Comment:
Can we delete __ms_?

Reply:
Wrong code on my side (missing args management)